README.md

```
@@ -468,6 +630,72 @@ For **DirectAPI scanning**, pass a single `RedactionSettings` object directly to
+For a complete list of overridable keys, extract them from the blinkid-ux AAR:
+```bash
+# Unzip the AAR and inspect res/values/values.xml
+unzip -p ~/.gradle/caches/modules-2/files-2.1/com.microblink/blinkid-ux/8000.0.0/*/blinkid-ux-8000.0.0.aar \
+  res/values/values.xml | grep 'name="mb_'
+```
+
+#### iOS
+
+Add a `BlinkID.strings` file to your `Runner` target for each locale (e.g. `en.lproj/BlinkID.strings`):
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)